### ISSUE TITLE
Cannot create torrent resume folder

### ISSUE THREAD

**Issue by ciasis** [wernight/docker-qbittorrent#25]
So after starting the container and fixing the _unable to write config file_ error, I have a new error that I don't know how to fix:

2021-07-01T17:19:54.350788722Z Cannot create torrent resume folder: "/home/qbittorrent/.local/share/qBittorrent/BT_backup"

Is this something inside the container or can I do anything about it?

**Comment by wernight** [comment]
I suppose that either you're running as another user than the default and/or you're mounting this or its parent directory. You need to ensure that the user you're running with has access to this directory for writing.

**Comment by ciasis** [comment]
Yes, I changed the PUID and GUID to my regular user I ran every docker container with. So should I keep the default user then?
